Test/floats/float4.bpl

```diff
@@ -1,19 +1,25 @@
 // RUN: %boogie -proverWarnings:1 "%s" > "%t"
 // RUN: %diff "%s.expect" "%t"
-procedure foo() returns (r : float8e24) {
+procedure foo() returns (r : float24e8) {
```

Comment:
Indentation of this file looks messy. Are you maybe using tabs??
Please clean this up.